pkgs/applications/video/kodi-packages/a4ksubtitles/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildKodiAddon, fetchFromGitHub, requests, vfs-libarchive  }:
+
+buildKodiAddon rec {
+  pname = "a4ksubtitles";
+  namespace = "service.subtitles.a4ksubtitles";
+  version = "2.3.0";
+
+  src = fetchFromGitHub {
+    owner = "a4k-openproject";
+    repo = "a4kSubtitles";
+    rev = "9855352a7ddd47f900f20601a366fd1ee83b9495";
```

```suggestion
    rev = "${namespace}/${namespace}-${version}";
```
https://github.com/a4k-openproject/a4kSubtitles/releases/tag/service.subtitles.a4ksubtitles/service.subtitles.a4ksubtitles-2.3.0
Releases seem to follow this format. More specific than just a random git hash, that we're not sure if it actually matches a tag or not.

---

Nice catch. I didn't notice that on the release page. Will investigate. Thanks.